src/consensus/aft/raft.h

```diff
@@ -395,12 +397,20 @@ namespace aft
       return get_latest_configuration().size();
     }
 
-    template <typename T>
+    template <typename T, typename H>
```

Comment:
Is it possible to not use a template type here? I would like to remote T in the future

Reply:
That's what I'd gone for at first, but it created a dependency cycle. A bunch of stuff has changed, let me try again.

Reply:
We can because of ConfigurableConsensus, done.